api/v1beta1/wso2is_types.go

```diff
@@ -217,207 +227,205 @@ type Keystore struct {
-	Parameters EmailOtpParameters `toml:"parameters,omitempty" json:"parameters,omitempty"`
+	Parameters EmailOtpParameters `json:"parameters,omitempty" toml:"parameters,omitempty"`
 }
 
 /* SMS OTP parameters */
 type SmsOtpParameters struct {
 	// +kubebuilder:default:="/smsotpauthenticationendpoint/smsotp.jsp"
-	SMSOTPAuthenticationEndpointURL string `toml:"SMSOTPAuthenticationEndpointURL,omitempty" json:"SMSOTPAuthenticationEndpointURL,omitempty"`
+	SMSOTPAuthenticationEndpointURL string `json:"SMSOTPAuthenticationEndpointURL,omitempty" toml:"SMSOTPAuthenticationEndpointURL,omitempty"`
 	// +kubebuilder:default:="/smsotpauthenticationendpoint/smsotpError.jsp"
-	SMSOTPAuthenticationEndpointErrorPage string `toml:"SMSOTPAuthenticationEndpointErrorPage,omitempty" json:"SMSOTPAuthenticationEndpointErrorPage,omitempty"`
+	SMSOTPAuthenticationEndpointErrorPage string `json:"SMSOTPAuthenticationEndpointErrorPage,omitempty" toml:"SMSOTPAuthenticationEndpointErrorPage,omitempty"`
 	// +kubebuilder:default:="/smsotpauthenticationendpoint/mobile.jsp"
-	MobileNumberRegPage string `toml:"MobileNumberRegPage,omitempty" json:"MobileNumberRegPage,omitempty"`
+	MobileNumberRegPage string `json:"MobileNumberRegPage,omitempty" toml:"MobileNumberRegPage,omitempty"`
 	// +kubebuilder:default:=true
-	RetryEnable bool `toml:"RetryEnable,omitempty" json:"RetryEnable,omitempty"`
+	RetryEnable bool `json:"RetryEnable,omitempty" toml:"RetryEnable,omitempty"`
 	// +kubebuilder:default:=true
-	ResendEnable bool `toml:"ResendEnable,omitempty" json:"ResendEnable,omitempty"`
+	ResendEnable bool `json:"ResendEnable,omitempty" toml:"ResendEnable,omitempty"`
 	// +kubebuilder:default:=true
-	BackupCode bool `toml:"BackupCode,omitempty" json:"BackupCode,omitempty"`
+	BackupCode bool `json:"BackupCode,omitempty" toml:"BackupCode,omitempty"`
 	// +kubebuilder:default:=true
-	SMSOTPEnableByUserClaim bool `toml:"SMSOTPEnableByUserClaim,omitempty" json:"SMSOTPEnableByUserClaim,omitempty"`
+	SMSOTPEnableByUserClaim bool `json:"SMSOTPEnableByUserClaim,omitempty" toml:"SMSOTPEnableByUserClaim,omitempty"`
 	// +kubebuilder:default:=false
-	SMSOTPMandatory bool `toml:"SMSOTPMandatory,omitempty" json:"SMSOTPMandatory,omitempty"`
+	SMSOTPMandatory bool `json:"SMSOTPMandatory,omitempty" toml:"SMSOTPMandatory,omitempty"`
 	// +kubebuilder:default:=true
-	CaptureAndUpdateMobileNumber bool `toml:"CaptureAndUpdateMobileNumber,omitempty" json:"CaptureAndUpdateMobileNumber,omitempty"`
+	CaptureAndUpdateMobileNumber bool `json:"CaptureAndUpdateMobileNumber,omitempty" toml:"CaptureAndUpdateMobileNumber,omitempty"`
```

Comment:
```suggestion
	CaptureAndUpdateMobileNumber bool `json:"captureAndUpdateMobileNumber,omitempty" toml:"captureAndUpdateMobileNumber,omitempty"`
```

The json key should be in camelCase right?